src/component/spinner.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {
+  ActivityIndicator,
+  StyleSheet,
+  View,
+  ART,
+} from 'react-native';
```

Comment:
Seems like there are currently linter errors in the build https://travis-ci.org/lightninglabs/lightning-app/jobs/394053652#L4509